api-tests/data/createperson.js

```diff
@@ -0,0 +1,46 @@
+const validPerson = {
```

Comment:
is it also possible to have the data files arranged to folders just as you do in the specs?